### PR TITLE
[SC-506] Audit(H-7): Asserted data unrelated to the assertedValue used in rewards

### PR DESCRIPTION
**Pull request by 0xNuggan**
## What has been done
When calling postAssertion, the KPIRewarder now sends the assertedValue as data.

### CODE REVIEW

**Comment by 0xNuggan** [comment]
I think there are some commit history shenanigans, but the diff looks good

File: test/modules/logicModule/LM_PC_KPIRewarder_v1.t.sol [hunk 1]
```diff
@@ -48,7 +48,7 @@ contract LM_PC_KPIRewarder_v1Test is ModuleTest {
 
     // Mock data for assertions
     bytes32 constant MOCK_ASSERTION_DATA_ID = "0x1234";
-    bytes32 constant MOCK_ASSERTION_DATA = "This is test data";
+    uint constant MOCK_ASSERTION_DATA = 100;
```
Comment (FHieser) [inline test/modules/logicModule/LM_PC_KPIRewarder_v1.t.sol]: Shouldnt this be named mock asserted Value ?

**Review by FHieser — APPROVED** [review]
LGTM
Added one comment that confused me in the tests

File: test/modules/logicModule/LM_PC_KPIRewarder_v1.t.sol (hunk 1 above)
Comment (0xNuggan) [reply test/modules/logicModule/LM_PC_KPIRewarder_v1.t.sol]: Good call. will update accordingly

**Comment by 0xNuggan** [comment]
tagging @marvinkruse  since it's high and we need 2 reviews

**Review by Zitzak — APPROVED** [review]

**Pull request merged**